Automattic/wp-calypso
# Automattic for Agencies: Update WP.com auth pages

Resolves https://github.com/Automattic/jetpack-genesis/issues/217

## Proposed Changes

- Adds logo for Automattic for Agencies
- Adds logo for A4A + WP 
- Updates signup page for A4A
- Updates login page for A4A

## Testing Instructions

Note: We will update the theme colors in another PR.

1) Open the Calypso live link
2) Append the URL with `/log-in?client_id=95928&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D95928%26redirect_uri%3Dhttps%253A%252F%252Fagencies.localhost%253A3000%252Fconnect%252Foauth%252Ftoken%253Fnext%253D%25252F%26scope%3Dglobal%26blog_id%3D0%26from-calypso%3D1`. `95928` is the dev OAuth client ID and verify that the page is updated as shown below:

<img width="729" alt="Screenshot 2024-02-15 at 10 10 18 AM" src="https://github.com/Automattic/wp-calypso/assets/10586875/e2e6d045-a1b0-4a38-a19f-a6d78f39da81">

3) Click the `Create an account` button -> Verify that the page is updated as shown below:

<img width="798" alt="Screenshot 2024-02-13 at 8 23 25 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/81d6dd9f-fba6-46d4-b653-e6d32e08acc8">

4) Repeat step 2 with OAuth client ID: Jetpack `/log-in?client_id=69040&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud.jetpack.com%252Fconnect%252Foauth%252Ftoken%253Fnext%253D%25252F%26scope%3Dglobal%26blog_id%3D0%26from-calypso%3D1`. Verify that the page has Jetpack Logo.

5. Repeat step 3 and verify that the Jetpack logo is shown

## Pre-merge Checklist
